rados/ioctx_octopus.go

```diff
@@ -1,5 +1,5 @@
-//go:build !nautilus
-// +build !nautilus
+//go:build octopus
+// +build octopus
```

Comment:
I don't think this is quite right as we actually expect the APIs in this file to be available *starting* in octopus, but also available in later ceph versions as well.  I'm pretty sure setting the build tag this way will cause it to be buildt only on octopus.